### PR TITLE
Remaining updates to Rasa 1.0

### PR DESCRIPTION
**Pull request by tabergma**
- update requirements
- remove not needed imports
- update Makefile
- use `rasa_sdk` instead of `rasa_core_sdk`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,36 +3,28 @@
...
 visualize:
-	python3 -m rasa_core.visualize -s data/core/ -d domain.yml -o story_graph.png
-
-train-online:
-	python -m rasa_core.train -u models/nlu/current/ --online --core models/dialogue/
+	rasa visualize --stories data/core/ --domain domain.yml --out story_graph.png
 
 evaluate-core:
-	python -m rasa_core.test --core models/dialogue -s data/core/ --fail_on_prediction_errors
+	rasa test core --model models/dialogue --stories data/core/ --fail-on-prediction-errors --quiet
```
Comment (akelad) [inline Makefile]: why quiet?

File: Makefile [hunk 2]
```diff
@@ -3,36 +3,28 @@
...
-	python3 -m rasa_nlu.train -c nlu_tensorflow.yml --fixed_model_name current --data data/nlu/ -o models --project nlu --verbose
-
-train-core:
-	python3 -m rasa_core.train -d domain.yml -s data/core -c policy.yml --debug -o models/dialogue
+train:
+	rasa train
 
 train-memo:
-	python -m rasa_core.train -d domain.yml -s data/core -c augmentedmemo-only.yml -o models/dialogue --augmentation 0
+	rasa train core --domain domain.yml --stories data/core --config augmentedmemo-only.yml --out models/dialogue --augmentation 0 --quiet
```
Comment (akelad) [inline Makefile]: why quiet?

File: Makefile (hunk 1 above)
Comment (tabergma) [reply Makefile]: travis failed and complaint that there is too much output

File: Makefile (hunk 2 above)
Comment (tabergma) [reply Makefile]: same

File: Makefile (hunk 1 above)
Comment (akelad) [reply Makefile]: hahaha that's a new one, it used to fail when there wasn't any output for a while

**Pull request merged**